### PR TITLE
Update exporter-toolkit

### PR DESCRIPTION
**Pull request by SuperQ**
Update to the latest exporter-toolkit
* Enables multi-listener and systemd socket activation.
* Bump Go to 1.19.
* Remove `PG_EXPORTER_WEB_LISTEN_ADDRESS` env var because this is now a repeatable flag.

Signed-off-by: SuperQ <superq@gmail.com>

### CODE REVIEW

File: cmd/postgres_exporter/main.go [hunk 1]
```diff
@@ -146,9 +145,8 @@ func main() {
 
 	http.HandleFunc("/probe", handleProbe(logger))
 
-	level.Info(logger).Log("msg", "Listening on address", "address", *listenAddress)
```
Comment (sysadmind) [inline cmd/postgres_exporter/main.go:149]: Does the exporter toolkit log the listen address? That seems useful for users. Cuts down on issues filed.

**Review by sysadmind — APPROVED** [review]
LGTM.

File: cmd/postgres_exporter/main.go (hunk 1 above)
Comment (SuperQ) [reply cmd/postgres_exporter/main.go:149]: Yup, the new version of the toolkit handles the logging, since the listen address can be called multiple times.

**Comment by yukha-dw** [comment]
Hello @SuperQ, is there any way to use custom port using environment variable after this change?

Thank you in advance!

**Pull request merged**